File: Competitive Programming/DAY-5/best_score.py
def bubblesort(arr):
    n=len(arr)
    for i in range(n-1):
        for j in range(0,n-i-1):
            if arr[j] > arr[j+1] : 
                arr[j], arr[j+1] = arr[j+1], arr[j] 
                
arr = [66, 57, 54, 53, 64, 52, 59] 
  
bubblesort(arr) 
  
print ("Best Score is:",arr[0]) 

File: Competitive Programming/DAY-5/best_score.cpp
#include <iostream>
using namespace std;
void best_score(int times[],int n)
{
    int temp;
    for(int i=0;i<n;i++)
    {
        for(int j=0;j<n-i-1;j++)
        {
            if(times[j]>times[j+1])
            {
                temp=times[j];
                times[j]=times[j+1];
                times[j+1]=temp;
            }
        }
    }
    cout<<"Best score is "<<times[0];
}

int main()
{
    int arr[]={66,57,54,53,64,52,59};
    int n=sizeof(arr)/sizeof(int);
    best_score(arr,n);
    return 0;
}
